.github/workflows/AutomateRelease.yml

```
@@ -0,0 +1,299 @@
+          jira_token: ${{ fromJSON(steps.secrets.outputs.vault).JIRA_TOKEN }}
+          ticket_key: ${{ needs.create_release_ticket.outputs.release_ticket_key }}
+          status: "Technical Release Done"
+          assignee: ${{ env.PM_EMAIL }}
+          use_sandbox: ${{ env.USE_JIRA_SANDBOX }}
+
+  unlock_master_branch:
+    name: Unlock master branch
+    needs: [ lock_master_branch, release_in_jira ]
+    if: always() && needs.lock_master_branch.result == 'success'
+    uses: ./.github/workflows/ToggleLockBranch.yml
+    permissions:
+      id-token: write
+
+  bump_versions:
+    name: Bump versions
+    needs: [ unlock_master_branch, release_in_jira ]
+    uses: ./.github/workflows/bump-versions.yaml
+    permissions:
+      contents: write # write for peter-evans/create-pull-request, read for actions/checkout
+      pull-requests: write # write for peter-evans/create-pull-request
+    with:
+      version: ${{ needs.release_in_jira.outputs.new_release_version }}-SNAPSHOT
+
+  update-integration-tickets:
+    name: Update Integration Tickets
+    runs-on: ubuntu-latest
+    needs: bump_versions
+    permissions:
+      contents: read
+      id-token: write
+    outputs:
+      sqs_ticket_key: ${{ steps.integration_update.outputs.sqs_ticket_key }}
+      sc_ticket_key: ${{ steps.integration_update.outputs.sc_ticket_key }}
+      sqs_ticket_url: ${{ steps.integration_update.outputs.sqs_ticket_url}}
+      sc_ticket_url: ${{ steps.integration_update.outputs.sc_ticket_url }}
+    steps:
+      - name: Get Jira Credentials from Vault
+        id: secrets
+        uses: SonarSource/vault-action-wrapper@v3
+        with:
+          secrets: |
+            development/kv/data/jira user | JIRA_USER;
+            development/kv/data/jira token | JIRA_TOKEN;
+
+      - name: Find and Update Tickets
+        id: integration_update
+        uses: SonarSource/release-github-actions/update-integration-tickets@09c557fb83722adbe2af2ca7e625324e15739362
+        with:
+          jira_user: ${{ fromJSON(steps.secrets.outputs.vault).JIRA_USER }}
+          jira_token: ${{ fromJSON(steps.secrets.outputs.vault).JIRA_TOKEN }}
+          release_ticket_key: "REL-3639"
+          sqs_fix_versions: ${{ github.event.inputs.sqs_fix_versions }}
+          use_sandbox: ${{ env.USE_JIRA_SANDBOX }}
+
+      - name: Echo Found Ticket Keys
+        run: |
+          echo "Found SQS integration ticket: ${{ steps.integration_update.outputs.sqs_ticket_key }}"
+          echo "Found SC integration ticket: ${{ steps.integration_update.outputs.sc_ticket_key }}"
+
+  update-analyzers:
+    name: Update Analyzers in SQS and SC
+    runs-on: ubuntu-latest
+    needs: [ update-integration-tickets, check_releasability ]
+    permissions:
+      id-token: write
+    outputs:
+      sqs_pr_url: ${{ steps.update_step_sqs.outputs.pr-url }}
+      sc_pr_url: ${{ steps.update_step_sc.outputs.pr-url }}
+    steps:
+      - name: get secrets
```

```suggestion
      - name: Get GitHub token from Vault
```
Biggest nitpick ever

---

😅